### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by ssk2**
Upgrade the version of Jenkins to the latest stable version, the Jenkins Mesos plugin and the Tomcat base image. 

The new plugin removes the need for users to specify a label for builds.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ WORKDIR /tmp
 # $CATALINA_HOME    is derived from the official Tomcat Dockerfile:
 #                   https://github.com/docker-library/tomcat/blob/df283818c1/8-jre8/Dockerfile
 #
-ENV JENKINS_WAR_URL https://updates.jenkins-ci.org/download/war/1.625.3/jenkins.war
+ENV JENKINS_WAR_URL https://updates.jenkins-ci.org/download/war/1.643/jenkins.war
```
Comment (rji) [inline Dockerfile]: Jenkins 1.643 is not an LTS release.

File: README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ quickly with Jenkins on the DCOS.
 
 ## Included in this repo
 Base packages:
-  * [Apache Tomcat][tomcat-home] 8.0.27
-  * [Jenkins][jenkins-home] 1.625.3 (LTS)
+  * [Apache Tomcat][tomcat-home] 8.0.30
+  * [Jenkins][jenkins-home] 1.643 (LTS)
```
Comment (rji) [inline README.md]: Same comment as above, 1.643 is not an LTS release.

**Comment by rji** [comment]
:-1: Jenkins 1.643 is a weekly release, not an LTS release. I don't think we should start adopting non-LTS releases for this project. If the Mesos plugin requires a newer version of Jenkins, we should wait to bump the plugin version until the next LTS release lands.

**Comment by ssk2** [comment]
Argh, my bad @rji. User error! I've removed that update.

**Comment by rji** [comment]
:+1: thanks @ssk2!

**Pull request merged**